attributes/default.rb

```diff
@@ -12,6 +12,8 @@
 default['formsender']['subdirectory'] = '' # add trailing slash if in a subdir
 
 default['formsender']['conf_email'] = 'support@osuosl.org'
+default['formsender']['conf_email_sup'] = 'support@osuosl.org'
```

Comment:
I suggest doing the following:

```
default['formsender']['emails']['support'] = 'support@osuosl.org'
default['formsender']['emails']['root'] = 'root@osuosl.org'
default['formsender']['emails']['default'] = 'support@osuosl.org'
```

You can then take advantage of this hash structure later in the template:

```erb
EMAIL = <%= '{' + node['formsender']['emails'].map {|k,v| "'#{k}': '#{v}'}.join(',') + '}' %>
```

And you construct the `EMAIL` dict for free no matter how many emails are in `node['formsender']['emails']`.